### PR TITLE
[powershell] Update to 7.0.3

### LINKED ISSUE #12849: [vcpkg_copy_tool_dependency] Call powershell function failed
**Host Environment**
- OS: Win10
- Compiler: revision

**To Reproduce**
`vcpkg install protobuf:x64-windows`

**Failure logs** 
Computing installation plan...
The following packages will be built and installed:
    protobuf[core]:x64-windows
Detecting compiler hash for triplet x64-windows...
Starting package 1/1: protobuf:x64-windows
Building package protobuf[core]:x64-windows...
Could not locate cached archive: C:\Users\Lollipop\AppData\Local\vcpkg\archives\83\83131633f57199177b05c9f2dec4699f08908e02.zip
-- Using cached D:/C/vcpkg/downloads/protocolbuffers-protobuf-31ebe2ac71400344a5db91ffc13c4ddfb7589f92.tar.gz
-- Extracting source D:/C/vcpkg/downloads/protocolbuffers-protobuf-31ebe2ac71400344a5db91ffc13c4ddfb7589f92.tar.gz
-- Applying patch fix-uwp.patch
-- Applying patch fix-android-log.patch
-- Applying patch fix-static-build.patch
-- Using source at D:/C/vcpkg/buildtrees/protobuf/src/dfb7589f92-61fbc4fa5f.clean
-- Configuring x64-windows
-- Building x64-windows-dbg
-- Building x64-windows-rel
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:72 (message):
    Command failed: D:/C/vcpkg/downloads/tools/powershell-core-6.2.1-windows/powershell.exe -noprofile -executionpolicy Bypass -nologo -file D:/C/vcpkg/scripts/buildsystems/msbuild/applocal.ps1 -targetBinary D:/C/vcpkg/packages/protobuf_x64-windows/tools/protobuf/protoc.exe -installedDir D:/C/vcpkg/packages/protobuf_x64-windows/bin
    Working Directory: D:/C/vcpkg
    Error code: -2146232797
    See logs for more information:
      D:\C\vcpkg\buildtrees\protobuf\copy-tool-dependencies-err.log

Call Stack (most recent call first):
  scripts/cmake/vcpkg_copy_tool_dependencies.cmake:27 (vcpkg_execute_required_process)
  scripts/cmake/vcpkg_copy_tool_dependencies.cmake:37 (search_for_dependencies)
  scripts/cmake/vcpkg_copy_tools.cmake:55 (vcpkg_copy_tool_dependencies)
  ports/protobuf/portfile.cmake:93 (vcpkg_copy_tools)
  scripts/ports.cmake:79 (include)

Error: Building package protobuf:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: protobuf:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.

**Additional context**
D:/C/vcpkg/packages/protobuf_x64-windows/tools/protobuf/protoc.exe
libprotobuf.dll and libprotoc.dll do not exist in D:/C/vcpkg/packages/protobuf_x64-windows/tools/protobuf

### PR DESCRIPTION
**Pull request by JackBoosY**
Update Powershell to 7.0.3 to solve the crash issue when calling the script `applocal.ps1` to find dependencies.

Fixes #12849.

### CODE REVIEW

File: ports/protobuf/CONTROL [hunk 1]
```diff
@@ -1,5 +1,6 @@
 Source: protobuf
-Version: 3.13.0
+Version: 3.13.0\
```
Comment (cenit) [inline ports/protobuf/CONTROL:2]: extra \

Comment (JackBoosY) [inline ports/protobuf/CONTROL:2]:
```suggestion
Version: 3.13.0
```

**Review by ras0219 — APPROVED** [review]
LGTM, though I would revert the changes to brotli and protobuf.

**Comment by BillyONeal** [comment]
I agree with @ras0219 ; we shouldn't force vcpkg upgrade to reinstall those ports over this. Rebased on master without those edits and will merge once CI comes back.

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**